README.rst

```diff
@@ -190,7 +190,7 @@ You must have `Docker installed <https://docs.docker.com/engine/installation/>`_
 
   **NOTE:** The need to use ``sudo`` for this step is dependent upon on your system configuration.
 
-4. There are 3 different options for running the QPC server.
+4. There are many different options for running the QPC server.
```

Comment:
If you view this, you will see that number 4 is in bold and italics. Makes it seem a little awkward compared to the rest of the doc.

Reply:
Agree.

Reply:
Super weird. It has been this way in master 😂 🤔

Reply:
Fixed